src/goto-instrument/code_contracts.cpp

```diff
@@ -1542,11 +1542,31 @@ goto_programt assigns_clauset::havoc_code(
   goto_programt havoc_statements;
   for(assigns_clause_targett *target : targets)
   {
+    //    if(target != NULL) goto x;
```

Comment:
May be you meant the following?
```suggestion
    //    if(target == NULL) goto z;
```